samples/cpp/common/utils/include/samples/args_helper.hpp

```diff
@@ -49,7 +51,7 @@ ov::element::Type getPrecision2(const std::string& value);
 
 template <class T>
 void printInputAndOutputsInfoShort(const T& network) {
-    std::cout << "Network inputs:" << std::endl;
+    slog::info << "Network inputs:" << slog::endl;
```

Comment:
Replaced standard output with log output as this function is only used in samples with logging

Reply:
README with output example should be modified as well